Packs/Phishing/Scripts/FindDuplicatePhishingIncidents/FindDuplicatePhishingIncidents.py

```diff
@@ -0,0 +1,349 @@
+    min_similarity = duplicate_incidents_df.iloc[-1]['similarity']
+    formatted_incident, headers = format_similar_incident(duplicate_incidents_df)
+    incident = 'incidents' if len(duplicate_incidents_df) > 1 else 'incident'
+    if max_similarity > min_similarity:
+        title = "Duplicate {} found with similarity {:.1f}%-{:.1f}%".format(incident, min_similarity * 100,
+                                                                            max_similarity * 100)
+    else:
+        title = "Duplicate {} found with similarity {:.1f}%".format(incident, max_similarity * 100)
+    message = tableToMarkdown(title,
+                              formatted_incident, headers)
+    if demisto.args().get('closeAsDuplicate', 'true') == 'true':
+        res = demisto.executeCommand("CloseInvestigationAsDuplicate", {
+            'duplicateId': most_similar_incident['id']})
+        if is_error(res):
+            return_error(res)
+        message += 'This incident (#{}) will be closed and linked to #{}.'.format(new_incident['id'],
+                                                                                  most_similar_incident['id'])
+    return_entry(message, duplicate_incidents_df, max_similarity)
+
+
+def create_new_incident():
+    return_entry('This incident is not a duplicate of an existing incident.')
+
+
+def format_similar_incident(duplicate_incidents_df):
+    incidents_list = duplicate_incidents_df.to_dict('records')
+    json_lists = []
+    status_map = {'0': 'Pending', '1': 'Active', '2': 'Closed', '3': 'Archive'}
+    for incident in incidents_list:
+        json_lists.append({'Id': "[%s](#/Details/%s)" % (incident['id'], incident['id']),
+                           'Name': incident['name'],
+                           'Status': status_map[str(incident.get('status'))],
+                           'Time': str(incident['created']),
+                           'Email From': incident.get(demisto.args().get('emailFrom')),
+                           'Text Similarity': "{:.1f}%".format(incident['similarity'] * 100),
+                           })
+    headers = ['Id', 'Name', 'Status', 'Time', 'Email From', 'Text Similarity']
+    return json_lists, headers
+
+
+def create_new_incident_low_similarity(duplicate_incidents_df):
+    message = '## This incident is not a duplicate of an existing incident.\n'
+    similarity = duplicate_incidents_df.iloc[0]['similarity']
+    if similarity > SIMILARITY_THRESHOLD - CLOSE_TO_SIMILAR_DISTANCE:
+        mask = duplicate_incidents_df['similarity'] >= SIMILARITY_THRESHOLD - CLOSE_TO_SIMILAR_DISTANCE
+        duplicate_incidents_df = duplicate_incidents_df[mask]
+        formatted_incident, headers = format_similar_incident(duplicate_incidents_df)
+        message += tableToMarkdown("Most similar incidents found", formatted_incident, headers=headers)
+        message += 'The threshold for considering 2 incidents as duplicate is a similarity ' \
```

Comment:
maybe we should change a bit this message, because here it's talking about similar incident, not necessary duplicate.
Also, maybe we can add a description to the script with to use 0.8 threshold to find similar incidents (not necessary duplicate)